python/ray/ml/train/data_parallel_trainer.py

```diff
@@ -348,3 +343,39 @@ def write_checkpoint(self, checkpoint: Dict):
     @property
     def latest_checkpoint_dir(self) -> Optional[Path]:
         raise NotImplementedError
+
+
+def _default_dataset_split_fn(
```

Comment:
Could we move this into the dataset spec file?

Reply:
This function is specific to `DataParallelTrainer`, not to `DatasetSpec` in general.